### PR TITLE
Docs: reword from 'experimental' to 'beta'

### PR DESCRIPTION
**Pull request by dyozie**
No longer using "experimental" in favor of the more generally-understood "Beta" for features still in development.

### CODE REVIEW

**Comment by dyozie** [comment]
Thanks @lisakowen - I'd somehow missed the markdown source for this change, but just committed those updates now.  

Overall I'm more in favor of clearly labeling Beta features rather than posting warnings about them.  Ideally we should just go with a (Beta) in the header, but a lot of our Beta features are smaller than header-level. I think going with a regular note rather than warning is more in line with getting more people to try out the features.

File: gpdb-doc/dita/admin_guide/wlmgmt.xml [hunk 1]
```diff
@@ -18,9 +18,8 @@
       4.4.73-5.1 or newer. <p>If you use RedHat 6 and the performance with resource groups is
         acceptable for your use case, upgrade your kernel to version 2.6.32-696 or higher to benefit
         from other fixes to the cgroups implementation. </p><p>SuSE 11 does not have a kernel
-        version that resolves this issue; resource groups are still considered to be an experimental
-        feature on this platform. <ph otherprops="pivotal">Resource groups are not supported on SuSE
-          11 for production use.</ph></p></note>
+        version that resolves this issue; resource groups are a Beta feature on this
```
Comment (litzell) [inline gpdb-doc/dita/admin_guide/wlmgmt.xml]: David Sharp commented on an earlier GPCC review: "The bug is fixed on SuSE 12. But we're still not supporting SuSE yet anyway."

File: gpdb-doc/dita/admin_guide/workload_mgmt_resgroups.xml [hunk 2]
```diff
@@ -524,9 +524,8 @@ rg_perseg_mem = ((RAM * (vm.overcommit_ratio / 100) + SWAP) * gp_resource_group_
         version 4.4.73-5.1 or newer. <p>If you use RedHat 6 and the performance with resource groups
           is acceptable for your use case, upgrade your kernel to version 2.6.32-696 or higher to
           benefit from other fixes to the cgroups implementation. </p><p>SuSE 11 does not have a
-          kernel version that resolves this issue; resource groups are still considered to be an
```
Comment (litzell) [inline gpdb-doc/dita/admin_guide/workload_mgmt_resgroups.xml]: Same comment about SuSE 12

File: gpdb-doc/dita/admin_guide/wlmgmt.xml (hunk 1 above)
Comment (dyozie) [reply gpdb-doc/dita/admin_guide/wlmgmt.xml]: Thanks - given that SuSE 12 is the target for support, I think we can remove this in master.

**Pull request merged**